tests/Backend/Workspaces/LegacyWorkspacesLoadTest.php

```diff
@@ -76,15 +76,23 @@ public function testWorkspaceLoadData()
         $input = array($mapping1, $mapping2);
 
         // test if job is created and listed
-        $initialJobs = $this->_client->listJobs();
         $runId = $this->_client->generateRunId();
         $this->_client->setRunId($runId);
```

Comment:
Proč se nepoužívá workspaceSapiClient všude a někde zůstal "starý" klient?

Reply:
Není potřeba

Reply:
A je potřeba aby ten starý klient vůbec existoval? Je to zbytečné riziko toho, že najednou máš dva klienty a u obou musíš třeba nastavit runid... přijde mi to divný, pokud to není potřeba.

Reply:
`workspaceSapiClient` je potřeba jen k tomu abych rozlišil který workspace patří k jakému testu

Reply:
> A je potřeba aby ten starý klient vůbec existoval?

Reply:
Tj. opět - není potřeba mě přesvědčovat k čemu to je. Úplně stačí, když odpovíš na to na co se ptám :)

Reply:
Tohle měnit nebudu.

Reply:
OK